src/plots/scatterPlot.ts

```diff
@@ -55,8 +55,7 @@ export class Scatter<X, Y> extends XYPlot<X, Y> {
         lightweightPlotEntity.index,
         lightweightPlotEntity.dataset);
 
-      // convert diameter into data space to be on the same scale as the scatter point position
-      lightweightPlotEntity.diameter = this._invertedPixelSize({ x: diameter, y: diameter });
+      lightweightPlotEntity.diameter = { x: diameter, y: diameter };
```

Comment:
@themadcreator It's the pixel size of the scatter point. Left a comment and updated so that it's just a number. On checking some more, exposed another bug with the calculation itself and opened a separate ticket to track, #3268.